docs-guides/source/opt-joint-compression.md

```diff
@@ -0,0 +1,335 @@
+Combining compression types
+===========================
+
+In previous sections ([palettization](opt-palettization), 
+[quantization](opt-quantization), [sparsity](opt-pruning) sections) 
+we considered how to apply the various compression 
+techniques to the weights/activations of the model independently. 
+In this section, we describe how these techniques can be combined, 
+which may be beneficial to get even more disk savings and latency improvements. 
+
+We first start by looking at how to take an uncompressed `mlpackage` 
+and get a joint compressed model by using the `ct.optimize.coreml.*` APIs. 
+As discussed in previous sections as well, this approach may or 
+may not yield a highly accurate model in some cases, however, 
+this is the best way to get the model in the desired format to test 
+out the expected disk size savings and  performance  (latency, runtime memory etc). 
+And once a model is obtained with desired performance characteristics, 
+a better accuracy model can be generated by applying the various data 
+based optimization methods available in `ct.optimize.torch.*`. 
+The latter is discussed via a few API code snippets in the section below. 
+
+## Combining compression types on an mlpackage 
+
+### Joint palettization and quantization 
+
+This means using a Look up table (LUT) whose values are of the 
+dtype INT8/UINT8 instead of Float16 which is the default. 
+This could be beneficial when combined with INT8 activations in
+speeding up inference. e.g. you could take a A16W16 model, 
+quantize the activations to get A8W16 model, and then quantize the 
+weights to a 4 bit LUT, with INT16 dtype, this will yield an A8W4 model, 
+where “W4” refers to a palettized weights with 8 bit LUT. 
```

Comment:
Might be better to make it more explicit because here the 8-bit and 4-bit could be confusing to new users. Maybe something like `“W4” refers to a palettized weights with a LUT that has 2**4 entries and each entry is INT8/UINT8.`?

Reply:
Good point, updated.